### ISSUE TITLE
two video missing in 8.06 Spring 2018. 

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-to-hugo#511]
@mbertrand @pdpinch There are two video missing in 8.06 Spring 2018. L13.4 and L13.5, which are easiest to see on this [page](https://ocw-studio.odl.mit.edu/sites/8-06-quantum-physics-iii-spring-2018/type/page/edit/775ae63a-83c3-7ea9-a86b-48e338f89741/). Let me know if there is a better way to report this or if this is the wrong place.

_Originally posted by @dseaton in https://github.com/mitodl/ocw-to-hugo/issues/459#issuecomment-1116455891_

**Comment by pdpinch** [comment]
@gumaerc is this similar to https://github.com/mitodl/ocw-data-parser/issues/193 ?

**Comment by gumaerc** [comment]
@dseaton : @pdpinch and I looked into this while chatting the other day and it doesn't seem to be related.  As far as we can tell, these videos don't even exist in the Plone exports.

**Comment by pdpinch** [comment]
Can we just allow the course author to set the YouTube ID in a case like this? The file already exists on YouTube.

**Comment by HussainTaj-arbisoft** [comment]
> Can we just allow the course author to set the YouTube ID in a case like this? The file already exists on YouTube.

I don't believe anything like that exists at the moment. I think we'd have to create a resource for the video page to be created. As suggested in https://github.com/mitodl/ocw-to-hugo/issues/511#issuecomment-1130082550

@pdpinch how many courses have this issue? If it's just these two videos, I could try to create the resources manually.

PS: @pdpinch could you please assign this ticket to me? I don't have the access to do it myself.
